static-site/content/blog/2024-06-12-new-resources-for-measles.md

```diff
@@ -0,0 +1,28 @@
+---
+author: "Nextstrain team"
+date: "2024-06-12"
+title: "New Resources for Measles Virus"
+---
+
+We now provide regularly updated phylogenetic monitoring of measles virus at [nextstrain.org/measles](https://nextstrain.org/measles). This site displays phylogenies generated using genomic data from NCBI GenBank, and is updated daily when new sequences are uploaded to NCBI. You can choose to display one of two different phylogenies based on the following data types:
+
+1. Full genome sequences
+1. 450bp region of the N gene ("N450")
+
+In both phylogenies, sequences are classified to genotype using the [nomenclature system established by the World Health Organization](https://www.who.int/publications/i/item/WER8709).
+
+As part of this work, we developed an [open-source tool for classifying measles genotypes](https://clades.nextstrain.org/?dataset-name=nextstrain/measles/N450/WHO-2012). This tool is a [Nextclade dataset](https://docs.nextstrain.org/projects/nextclade/en/stable/user/datasets.html), which allows users to drag-and-drop sequences onto a web browser to obtain genotype classifications. It also outputs sequence quality control metrics and displays user sequences on a phylogenetic tree.
+
+We also provide daily-updated links where all NCBI GenBank measles [sequences](https://data.nextstrain.org/files/workflows/measles/sequences.fasta.zst) and [metadata](https://data.nextstrain.org/files/workflows/measles/metadata.tsv.zst) can be downloaded. These currently include over 23,000 sequences, of which ~1,000 are full genome sequences, and ~19,000 include the N450 region.
+
+With measles cases currently increasing globally, we expect these tools to be useful resources for tracking the spread of this viral pathogen. This work is made possible by the open sharing of genetic data by research groups from all over the world. We gratefully acknowledge their contributions.
```

Comment:
> With measles cases currently increasing globally

I wonder if it's worth finding and linking an authoritative source here.

Reply:
Done in [5bddcd7](https://github.com/nextstrain/nextstrain.org/pull/911/commits/5bddcd7659cfa2d55712dc2465d39c04bd18291f)